src/index.js

```diff
@@ -1,32 +1,43 @@
 var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 
-var API = 'https://rickandmortyapi.com/api/character/';
+var API = "https://rickandmortyapi.com/api/character/";
 var xhttp = new XMLHttpRequest();
 
-function fetchData(url_api, callback) {
-  xhttp.onreadystatechange = function (event) {
-    if (xhttp.readyState === '4') {
-      if (xhttp.status == 200)
-        callback(null, xhttp.responseText);
-      else return callback(url_api);
-    }
-  };
-  xhttp.open('GET', url_api, false);
-  xhttp.send();
+var fetchData = function(url_api, data) {
```

Comment:
Eliminar la variable data no la estas utilizando en tu función

```suggestion
var fetchData = function(url_api) {
```